### ISSUE TITLE
404

### ISSUE THREAD

**Issue by charlesYun** [alibaba/hooks#1009]
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npm.taobao.org/@types/jest/-/jest-26.0.15.tgz - [not_found] document not found
npm ERR! 404 
npm ERR! 404  '@types/jest@https://registry.npm.taobao.org/@types/jest/-/jest-26.0.15.tgz' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

**Comment by brickspert** [comment]
用 yarn 试试

**Comment by brickspert** [comment]
拉下最新代码重新试试

**Comment by liuxuanjs** [comment]
我拉的最新代码，npm run init 的时候 还是会报这个错

**Comment by proc07** [comment]
@brickspert   我这边也是 npm run init  报错，目前用 yarn 命令可以运行

**Comment by brickspert** [comment]
Hello：

这个问题我们已经在 v3.0 版本彻底修复，目前 v3.0 版本已经发布 alpha 版本。

欢迎试用。

文档：https://ahooks-next.surge.sh/
源码：https://github.com/alibaba/hooks/tree/next

安装最新分支
```
$ npm install --save ahooks@next
# or
$ yarn add ahooks@next
```

We have fixed this problem in the v3.0 version, and the alpha has been released. 

Welcome to try.

Doc：https://ahooks-next.surge.sh/
Source Code：https://github.com/alibaba/hooks/tree/next

Install v3.0

```
$ npm install --save ahooks@next
# or
$ yarn add ahooks@next
```
